lib/geocoder/lookups/ban_data_gouv_fr.rb

```diff
@@ -5,7 +5,7 @@ module Geocoder::Lookup
   class BanDataGouvFr < Base
 
     def name
-      "Base Adresse Nationale Française"
+      "Base Adresse Nationale Francaise"
```

Comment:
Ok I will do this, I also spotted this issue in other files with different characters. 

I only need to insert `# encoding: utf-8` on top of each file ? 
Is there a way to make files inherit this encoding ?
Will it be an issue in fixtures files (which also contain utf-8 chars) ?

Thanks for your assistance